api/v1alpha3/conversion.go

```diff
@@ -251,6 +252,24 @@ func Convert_v1alpha4_Instance_To_v1alpha3_Instance(in *v1alpha4.Instance, out *
 	return autoConvert_v1alpha4_Instance_To_v1alpha3_Instance(in, out, s)
 }
 
+// Convert_v1alpha3_Network_To_v1alpha4_NetworkStatus is an autogenerated conversion function.
+func Convert_v1alpha3_Network_To_v1alpha4_NetworkStatus(in *Network, out *v1alpha4.NetworkStatus, s apiconversion.Scope) error {
+	out.SecurityGroups = *(*map[v1alpha4.SecurityGroupRole]v1alpha4.SecurityGroup)(unsafe.Pointer(&in.SecurityGroups))
+	if err := Convert_v1alpha3_ClassicELB_To_v1alpha4_ClassicELB(&in.APIServerELB, &out.APIServerELB, s); err != nil {
+		return err
+	}
+	return nil
+}
+
+// Convert_v1alpha4_NetworkStatus_To_v1alpha3_Network is an autogenerated conversion function.
+func Convert_v1alpha4_NetworkStatus_To_v1alpha3_Network(in *v1alpha4.NetworkStatus, out *Network, s apiconversion.Scope) error {
+	out.SecurityGroups = *(*map[SecurityGroupRole]SecurityGroup)(unsafe.Pointer(&in.SecurityGroups))
+	if err := Convert_v1alpha4_ClassicELB_To_v1alpha3_ClassicELB(&in.APIServerELB, &out.APIServerELB, s); err != nil {
+		return err
+	}
+	return nil
+}
+
```

Comment:
Why was this copied from the autogenerated conversions?

Reply:
shall we directly set it as out.SecurityGroups = in.SecurityGroups ?

Reply:
Let's catch up tomorrow morning. Noticed a few other errors in the conversions that aren't related to this PR that need fixing first.

Reply:
Had a check of this. Can confirm there isn't another way to do this. Can we update the comment about it being an "autogenerated conversion function".

for the specific comment, i would put something like
`Convert_v1alpha4_NetworkStatus_To_v1alpha3_Network is based on the autogenerated function and handles the renaming of the Network struct to NetworkStatus`

If you can, can you also get rid of the "is an autogenerated conversion function" text from the other comments in the conversion.go file, as it's not accurate.

Reply:
sure I will update the comments